### PR TITLE
Update the userstore.yaml

### PR DESCRIPTION
**Pull request by sachiniWettasinghe**
Resolves https://github.com/wso2/product-is/issues/9546

This PR is to update the userstore.yaml with [1].

[1] https://github.com/wso2/identity-api-server/blob/master/components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/resources/userstore.yaml

### CODE REVIEW

File: en/docs/develop/restapis/userstore.yaml [hunk 1]
```diff
@@ -65,19 +61,18 @@ paths:
     get:
       tags:
         - User Store
-      summary: Retrieve or list the configured secondary user stores.
+      summary: Retrieve or List the configured secondary user stores.
```
Comment (pulasthi7) [inline en/docs/develop/restapis/userstore.yaml:64]:
```suggestion
      summary: Retrieve or list the configured secondary user stores.
```

File: en/docs/develop/restapis/userstore.yaml [hunk 2]
```diff
@@ -219,20 +209,19 @@ paths:
               $ref: '#/components/schemas/UserStoreReq'
               # But not allowing to change the typeName and name in the PUT request.
         description: >-
-          The secondary user store values that need to be edited for a
-          given domain ID.
+          The secondary user store values which are need to be edited of the
```
Comment (pulasthi7) [inline en/docs/develop/restapis/userstore.yaml:212]:
```suggestion
          The secondary user store values which are needed to be edited of the
```

File: en/docs/develop/restapis/userstore.yaml [hunk 3]
```diff
@@ -219,20 +209,19 @@ paths:
...
+          given domain id.
     patch:
       tags:
         - User Store
-      summary: Patch the secondary user store by its domain ID.
+      summary: Patch the secondary user store by it's domain id.
```
Comment (pulasthi7) [inline en/docs/develop/restapis/userstore.yaml:217]:
```suggestion
      summary: Patch the secondary user store by its domain id.
```

File: en/docs/develop/restapis/userstore.yaml [hunk 4]
```diff
@@ -219,20 +209,19 @@ paths:
...
       operationId: patchUserStore
       description: >
         This API provides the capability to update the secondary user store's
-        property using a patch request made based on its domain ID. <br>
-        <b>Permission required:</b><br>
-        */permission/admin/manage/identity/userstore/config/update<br>
-        <b>Scope required:</b><br>
-        * internal_userstore_update
+        property using patch request by using it's domain id.
```
Comment (pulasthi7) [inline en/docs/develop/restapis/userstore.yaml:221]:
```suggestion
        property using patch request by using its domain id.
```

File: en/docs/develop/restapis/userstore.yaml [hunk 5]
```diff
@@ -363,48 +345,55 @@ paths:
...
+      description: number of records to skip for pagination
       schema:
         type: integer
         format: int32
     filterQueryParam:
       in: query
       name: filter
       required: false
-      description: Condition to filter the retrieval of records.
+      description: Condition to filter the retrival of records.
```
Comment (pulasthi7) [inline en/docs/develop/restapis/userstore.yaml:382]:
```suggestion
      description: Condition to filter the retrieval of records.
```

File: en/docs/develop/restapis/userstore.yaml [hunk 6]
```diff
@@ -363,48 +345,55 @@ paths:
...
       schema:
         type: string
     sortQueryParam:
       in: query
       name: sort
       required: false
-      description: Define the order in which the retrieved records should be sorted.
+      description: Define the order how the retrieved records should be sorted.
```
Comment (pulasthi7) [inline en/docs/develop/restapis/userstore.yaml:389]:
```suggestion
      description: Define the order of how the retrieved records should be sorted.
```

File: en/docs/develop/restapis/userstore.yaml [hunk 7]
```diff
@@ -592,38 +577,31 @@ components:
         name:
           type: string
           example: JDBC-SECONDARY
-          description: Domain name of the secondary user store
+          description: domain name of the secondary user store
```
Comment (pulasthi7) [inline en/docs/develop/restapis/userstore.yaml:580]: Shouldn't this start with a capital letter? There is an inconsistency when compared with other similar descriptions

**Review by sherenewso2 — APPROVED** [review]

**Pull request merged**